content/manuals/desktop/networking.md

```diff
@@ -184,3 +184,29 @@ to give the container direct access to the network stack of the host.
 
 See the [run command](/reference/cli/docker/container/run.md) for more details on
 publish options used with `docker run`.
+
+### IPv4-only, Dual-stack and IPv6-only (beta)
```

Comment:
What is the default?

Reply:
I added a note about the default mode in the first paragraph.